index.html

```diff
@@ -48,7 +48,7 @@
             </div>
             <nav class="navigation">
```

Comment:
Great job @godybuoy 🤝

- [ ] Kindly follow the design guidelines as required in the project requirements. You may align your navbar elements properly so that your design may replicate the design provided.
- Your design
![your-navbar](https://user-images.githubusercontent.com/63917725/235324182-eb78cd0e-9dd9-4431-9318-0d6a7a090975.PNG)
- Given design
![given navbar](https://user-images.githubusercontent.com/63917725/235324198-9aa12f6b-54a7-4e60-80b8-88327af1b00b.PNG)